code/modules/uplink/uplink_items.dm

```diff
@@ -37,29 +37,29 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 
 			if(!filtered_uplink_items[category])
 				filtered_uplink_items[category] = list()
-			filtered_uplink_items[category][item] = I
+			filtered_uplink_items[category][item] = new I.type()
```

Comment:
Why is this necessary?

Reply:
So that limited_stock is not shared between items on different uplinks